lib/classes/PluginManager.js

```diff
@@ -61,8 +82,12 @@ class PluginManager {
     });
 
     if (hooks.length === 0) {
-      throw new this.serverless.classes.Error('The command you entered was not found. ' +
-        'Did you spell it correctly?');
+      const errorMessage =
+     `
```

Comment:
This newline (and also the one below) can be removed once the automatic newline addition is done in the Error class.